### ISSUE TITLE
CGRPCZlib podspec have wrong LICENSE filename

### ISSUE THREAD

**Issue by Shedward** [grpc/grpc-swift#789]
### New Issue Checklist

- [x] Updated grpc-swift to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/grpc/grpc-swift/blob/master/CONTRIBUTING.md)
- [x] I searched for [existing GitHub issues](https://github.com/grpc/grpc-swift/issues)(especially, there are many [resolved issues related to SSL](https://github.com/grpc/grpc-swift/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aclosed+ssl))

### Issue Description
Current [CGRPCZlib podspec](https://github.com/grpc/grpc-swift/blob/master/scripts/build_podspecs.py) have wrong license filename. 
In podspec is `LICENSE.txt` while actual license file is `LICENSE`.
It creates bunch of warning on `pod install`
```
[!] Unable to read the license file `LICENSE.txt` for the spec `CGRPCZlib (1.0.0-alpha.11)
```
`build_podspecs.py` template seems to be correct, so please update podspec for CGRPCZlib.

### Environment

<details>
  <pre>
  
| Key                           | Value                                                         |
| ----------------------------- | --------------------------------------------------------------|
| gRPC-Swift Version      |   1.0.0-alpha.11                                                       |
| gRPC-Swift Version  |  1.0.0-alpha.11                                                         |
| CocoaPods Version  |  1.9.1                                                                         |
  </pre>
</details>

**Comment by MrMage** [comment]
Feel free to file a pull request :-)

**Comment by glbrntt** [comment]
@MrMage I don't think we need a PR; the script [does the right thing](https://github.com/grpc/grpc-swift/blob/master/CGRPCZlib.podspec), we just need to run the script to upload the new podspecs.

**Comment by MrMage** [comment]
> @MrMage I don't think we need a PR; the script [does the right thing](https://github.com/grpc/grpc-swift/blob/master/CGRPCZlib.podspec), we just need to run the script to upload the new podspecs.

Sorry, I was not aware of that. I have tried to update the Pod now with `python3 scripts/build_podspecs.py -u 1.0.0-alpha.11` and received the following log output:

```
Building Podspec for CGRPCZlib
-----------------------------------------------------------
    Writing to /Users/daniel/Documents/Xcode/Mac/grpc-swift-tmp/CGRPCZlib.podspec 
Updating spec repo `trunk`

CocoaPods 1.9.1 is available.
To update use: `gem install cocoapods`

For more information, see https://blog.cocoapods.org and the CHANGELOG for this version at https://github.com/CocoaPods/CocoaPods/releases/tag/1.9.1

    Publishing CGRPCZlib.podspec
Cloning spec repo `/users/daniel/documents/xcode/mac/grpc-swift-tmp/cgrpczlib.podspec-1` from `/Users/daniel/Documents/Xcode/Mac/grpc-swift-tmp/CGRPCZlib.podspec`
[!] Unable to find the `/Users/daniel/Documents/Xcode/Mac/grpc-swift-tmp/CGRPCZlib.podspec` repo. If it has not yet been cloned, add it via `pod repo add`.


Building Podspec for gRPC-Swift
-----------------------------------------------------------
    Writing to /Users/daniel/Documents/Xcode/Mac/grpc-swift-tmp/gRPC-Swift.podspec 
Updating spec repo `trunk`

CocoaPods 1.9.1 is available.
To update use: `gem install cocoapods`

For more information, see https://blog.cocoapods.org and the CHANGELOG for this version at https://github.com/CocoaPods/CocoaPods/releases/tag/1.9.1

    Publishing gRPC-Swift.podspec
Cloning spec repo `/users/daniel/documents/xcode/mac/grpc-swift-tmp/grpc-swift.podspec-1` from `/Users/daniel/Documents/Xcode/Mac/grpc-swift-tmp/gRPC-Swift.podspec`
[!] Unable to find the `/Users/daniel/Documents/Xcode/Mac/grpc-swift-tmp/gRPC-Swift.podspec` repo. If it has not yet been cloned, add it via `pod repo add`.
```

@Jake-Prickett do you have any idea what I could do to fix this?

**Comment by MrMage** [comment]
Okay, @glbrntt pointed out replacing "repo" with "trunk" in one place in the script, that seems to have worked. On it...

**Comment by Jake-Prickett** [comment]
Hello! Were you able to resolve the issue with the `trunk` change @MrMage? We are seeing these warnings well in my current project, from when I initially pushed the gRPC-Swift & CGRPCZlib Pods.

The warning is resolved within the script (as you've seen) and is waiting on a new Release to push the pod. I was under the impression that you could not overwrite a Pod? It would require deleting the Pod, and re-pushing which could cause issues for consumers. (I could be wrong 😄)

**Comment by glbrntt** [comment]
That seems to be the case @Jake-Prickett -- we'd have to delete and push it. No idea if that would cause issues or not. I'm okay with just tagging a new release.

**Comment by MrMage** [comment]
@Jake-Prickett FYI, I am seeing these errors now:

```
Building Podspec for CGRPCZlib
-----------------------------------------------------------
    Writing to /Users/daniel/Documents/Xcode/Mac/grpc-swift-tmp/CGRPCZlib.podspec 
Updating spec repo `trunk`
    Publishing CGRPCZlib.podspec
Updating spec repo `trunk`
Validating podspec
 -> CGRPCZlib (1.0.0-alpha.11)
    - NOTE  | xcodebuild:  note: Using new build system
    - NOTE  | xcodebuild:  note: Planning build
    - NOTE  | xcodebuild:  note: Constructing build description
    - NOTE  | xcodebuild:  warning: Skipping code signing because the target does not have an Info.plist file. (in target 'App')

[!] Unable to accept duplicate entry for: CGRPCZlib (1.0.0-alpha.11)


Building Podspec for gRPC-Swift
-----------------------------------------------------------
    Writing to /Users/daniel/Documents/Xcode/Mac/grpc-swift-tmp/gRPC-Swift.podspec 
Updating spec repo `trunk`
    Publishing gRPC-Swift.podspec
Updating spec repo `trunk`
Validating podspec
 -> gRPC-Swift (1.0.0-alpha.11)
    - NOTE  | xcodebuild:  note: Using new build system
    - NOTE  | [iOS] xcodebuild:  note: Planning build
    - NOTE  | [iOS] xcodebuild:  note: Constructing build description
    - NOTE  | [iOS] xcodebuild:  warning: Skipping code signing because the target does not have an Info.plist file. (in target 'App')
    - ERROR | [OSX] unknown: Encountered an unknown error (CocoaPods could not find compatible versions for pod "CGRPCZlib":
  In Podfile:
    gRPC-Swift (from `/Users/daniel/Documents/Xcode/Mac/grpc-swift-tmp/gRPC-Swift.podspec`) was resolved to 1.0.0-alpha.11, which depends on
      CGRPCZlib (= 1.0.0-alpha.11)

Specs satisfying the `CGRPCZlib (= 1.0.0-alpha.11)` dependency were found, but they required a higher minimum deployment target.) during validation.

[!] The spec did not pass validation, due to 1 error.

[!] Unable to read the license file `LICENSE.txt` for the spec `CGRPCZlib (1.0.0-alpha.11)`

[!] Unable to read the license file `LICENSE.txt` for the spec `CGRPCZlib (1.0.0-alpha.11)`
```

To me, that sounds like we would need to tag a .12 release and then push that.

**Comment by glbrntt** [comment]
A new podspec has been published (#793); this should be fixed now.
